F/FastJet/build_tarballs.jl

```diff
@@ -28,9 +28,21 @@ if [[ "${target}" == *-mingw* ]]; then
     #   libtool: warning: undefined symbols not allowed in x86_64-w64-mingw32 shared libraries; building static only
     FLAGS+=(LDFLAGS="-no-undefined")
 fi
-./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --disable-auto-ptr
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --disable-auto-ptr --disable-static
 make -j ${nprocs} "${FLAGS[@]}"
 make install
+tail -n 340 COPYING > LICENSE
+install_license LICENSE
+cp include/fastjet/config_win.h ${includedir}/fastjet/
+
+cd $WORKSPACE/srcdir/fjcontrib-*/
+for name in ClusteringVetoPlugin ConstituentSubtractor EnergyCorrelator FlavorCone GenericSubtractor JetCleanser JetFFMoments JetsWithoutJets LundPlane Nsubjettiness QCDAwarePlugin RecursiveTools ScJet SoftKiller SubjetCounting ValenciaPlugin VariableR
+do
+    cd $name
+    rm example*
+    c++ -fPIC -shared -I${includedir} -O3 -Wall ${LDFLAGS} [A-Z]*.cc -o "${libdir}/lib${name}.${dlext}" -lfastjet -lfastjettools
```

Comment:
As a general remark, libraries must be mentioned _after_ they are required.  Unless asked to do otherwise, the linker usually keeps only the symbols needed so far, so if you require a library before any of its symbol is actually asked for, this is like not using that library at all.  For this reason, the order of libraries matter a lot, especially on Windows where undefined symbols are not allowed.  See for example https://wiki.gentoo.org/wiki/Project:Quality_Assurance/As-needed

Reply:
I probably knew that at some point, but forgot. Thanks for the link.